tokio/src/runtime/builder.rs

```diff
@@ -1245,8 +1245,30 @@ impl Builder {
         ///     .unwrap();
         /// ```
         ///
+        /// Configure a `LogHistogram` with similar behavior to [`HistogramScale::Log`]
```

Comment:
This sentence is confusing. I'm guessing `HistogramScale::Log` is the old implementation?

Reply:
yes clarified